src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -195,6 +195,10 @@ public void refreshAdditionalCommands() {
         getAdditionalCommands().forEach(this::defineCommand);
     }
 
+    public void defineCommand(String commandName, CommandInfo info) {
```

Comment:
cannot we just call super.defineCommand from refreshAdditionalCommands and avoid adding an additional method?

Reply:
we need to invoke this method from `AppiumDriver.java`, and as it's in another package, this method should be `public`, so we need to override it.

as for `refreshAdditionalCommands` it iterates through all additional commands and adds them to codec, this functionality is needed when a new codec is set (see line #183 of this class)

Reply:
you mean 

```
public void refreshAdditionalCommands() {
     getAdditionalCommands().forEach(super::defineCommand);
}
```

won't work?

Reply:
> refreshAdditionalCommands will not work in AppiumDriver

why it won't work? it calls the same defineCommand API.

Reply:
makes sense